src/components/AppNavigationSettings/AppNavigationSettings.vue

```diff
@@ -79,6 +79,11 @@ export default {
 }
 </script>
 <style lang="scss" scoped>
+
+#app-settings-header .settings-button {
+	padding-left: 50px;
+}
+
```

Comment:
I would argue the opposite, should be fixed in files, no?
What is the AppNavigationItem using?

Reply:
This is not Files, this is Tasks and Calendar and literally every other app which uses AppNavigationItem. These 6 px here:
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/AppNavigationItem/AppNavigationItem.vue#L520
add to the 44 px icon width, which make it 50 px in total.

Reply:
This was added in this commit https://github.com/nextcloud/nextcloud-vue/commit/5e7e3950d30e7188c55a3f414767fe16bdccd94d#